src/specular/sequencing/config.rs

```diff
@@ -1,6 +1,6 @@
 use ethers::types::H160;
 
-use crate::config;
+use crate::config::{Config as MagiConfig, SystemConfig as MagiSystemConfig};
```

Comment:
Is `SystemConfig` over loaded? I think we can skip renaming, unless there is a conflict. (I agree on the `Config as MagiConfig` though)

Reply:
NVM. Yes there is a conflict. Ignore this.